lib/FlashMem/FlashMem.h

```diff
@@ -140,6 +146,42 @@ namespace Flash
      */
     bool saveString(const uint8_t &address, const uint8_t &maxLength, const String &input);
 
+    /**
+     *  Retrieves an Unsigned Integer from the EEPROM.
+     *
+     *  @param[in] address Address where the String is saved.
+     *  @param[out] value Buffer to save the Integer to.
+     *  @return If Integer succesfully retrieved from EEPROM, returns True. 
+     *          Otherwise False.
+     */
+    bool fetchInt(const uint8_t &address, uint8_t &value);
+
+    /**
+     *  Saves an Unsigned Integer in the EEPROM.
+     *
+     *  @param[in] address Address where the String will be saved.
+     *  @param[in] value String to save in EEPROM.
+     *  @return If integer written in EEPROM, returns True. Otherwise False.
+     */
+    bool saveInt(const uint8_t &address, const uint8_t &value);
+
+    /**
+     *  Retrieves the Number of Groups from the EEPROM.
+     * 
+     *  @param[out] groups Buffer to save the Number of Groups to.
+     *  @return If the value is succesfully retrieved from EEPROM, returns True. 
+     *          Otherwise False.
```

Comment:
Don't write about the python True/False here.

Reply:
What would be the correct way to do it? Should I write `true/false` instead? Or is my whole `@return` description wrong?

Reply:
Changed the lowercase :) 

Then it would make sense to move this method to the Competition class, so the FlashMem Module has nothing to do with the Groups